### PR TITLE
Xtz polls

### LINKED ISSUE #763: XTZ-weighted offchain poll - BACKEND
**The task is to allow users to vote on an offchain poll with their XTZ balance instead of the designated governance token of the DAO.** 

When creating a new offchain poll, we store another bool `isXTZ` that defaults to `false` unless the user explicitely sets it to `true.`  

If `isXTZ` is `true` for any given poll, everything else remains the same, but for the vote action, the app queries a different endpoint on tzkt.io that returns the user's XTZ balance instead of the balance of the governance token. 

This task also involves a bit of research to figure out what endpoint to call on tzkt in order to get back the XTZ balance of the user.

### PR DESCRIPTION
**Pull request by fabiolalombardim**
Closes #763 #764

### CODE REVIEW

File: src/modules/lite/explorer/pages/ProposalDetails/index.tsx [hunk 1]
```diff
@@ -85,14 +87,19 @@ export const ProposalDetails: React.FC<{ id: string }> = ({ id }) => {
     }
   })
 
+  const dataForBack = {
+    isXTZ: poll?.isXTZ,
```
Comment (Man-Jain) [inline src/modules/lite/explorer/pages/ProposalDetails/index.tsx:91]: Don't think we need this now.

File: src/modules/lite/explorer/components/VoteDetails.tsx [hunk 2]
```diff
@@ -174,17 +187,17 @@ export const VoteDetails: React.FC<{ poll: Poll | undefined; choices: Choice[];
             justifyContent={isMobileSmall ? "flex-start" : "flex-end"}
           >
             <Typography color="textPrimary" variant="body1">
-              {nFormatter(calculateProposalTotal(choices, tokenData?.decimals), 1)}
+              {numbro(calculateProposalTotal(choices, isXTZ ? 0 : tokenData?.decimals)).format(formatConfig)}
```
Comment (Man-Jain) [inline src/modules/lite/explorer/components/VoteDetails.tsx:190]: XTZ decimals is 18 by default. Unless it's a different case with the data we're getting in this case from the API or Homebase Backend.

File: src/modules/lite/explorer/components/VoteDetails.tsx [hunk 3]
```diff
@@ -174,17 +187,17 @@ export const VoteDetails: React.FC<{ poll: Poll | undefined; choices: Choice[];
...
             </Typography>
             <Typography color="textPrimary" variant="body1">
               {poll?.tokenSymbol}
             </Typography>
             <Typography color="textPrimary" variant="body1">
               (
               {getTreasuryPercentage(
-                calculateProposalTotal(choices, tokenData?.decimals),
+                calculateProposalTotal(choices, isXTZ ? 0 : tokenData?.decimals),
```
Comment (Man-Jain) [inline src/modules/lite/explorer/components/VoteDetails.tsx:198]: Same XTZ Decimals - 18

File: src/modules/lite/explorer/components/VoteDetails.tsx [hunk 4]
```diff
@@ -110,7 +119,11 @@ export const VoteDetails: React.FC<{ poll: Poll | undefined; choices: Choice[];
                   {choice && choice.walletAddresses ? (
                     <Typography color="textPrimary" variant="body2">
                       {choice.walletAddresses.length} Voters -{" "}
-                      {nFormatter(calculateChoiceTotal(choice.walletAddresses, tokenData?.decimals), 1)}{" "}
+                      {!isMobile
+                        ? nFormatter(calculateChoiceTotal(choice.walletAddresses, isXTZ ? 0 : tokenData?.decimals), 1)
```
Comment (Man-Jain) [inline src/modules/lite/explorer/components/VoteDetails.tsx:123]: Same XTZ Decimals - 18

File: src/modules/lite/explorer/components/VoteDetails.tsx [hunk 5]
```diff
@@ -110,7 +119,11 @@ export const VoteDetails: React.FC<{ poll: Poll | undefined; choices: Choice[];
...
+                        : numbro(calculateChoiceTotal(choice.walletAddresses, isXTZ ? 0 : tokenData?.decimals)).format(
```
Comment (Man-Jain) [inline src/modules/lite/explorer/components/VoteDetails.tsx:124]: Same XTZ Decimals - 18

**Review by Man-Jain — CHANGES_REQUESTED** [review]
Left some comments

File: src/modules/lite/explorer/pages/ProposalDetails/index.tsx (hunk 1 above)
Comment (fabiolalombardim) [reply src/modules/lite/explorer/pages/ProposalDetails/index.tsx]: fixed!

File: src/modules/lite/explorer/components/VoteDetails.tsx (hunk 2 above)
Comment (fabiolalombardim) [reply src/modules/lite/explorer/components/VoteDetails.tsx:190]: fix!

File: src/modules/lite/explorer/components/VoteDetails.tsx (hunk 4 above)
Comment (fabiolalombardim) [reply src/modules/lite/explorer/components/VoteDetails.tsx:123]: fix! - Decimals 6

File: src/modules/lite/explorer/components/VoteDetails.tsx (hunk 5 above)
Comment (fabiolalombardim) [reply src/modules/lite/explorer/components/VoteDetails.tsx:124]: fix! - Decimals 6

**Review by Man-Jain — APPROVED** [review]

**Pull request merged**